pkg/binding/asker/survey_asker.go

```diff
@@ -61,3 +110,34 @@ func (o *Survey) AskBindAsFiles() (bool, error) {
 	}
 	return bindAsFile, nil
 }
+
+func (o *Survey) SelectCreationOption() ([]CreationOption, error) {
+	options := []int{}
+	prompt := &survey.MultiSelect{
+		Message: "Check(with Space Bar) one or more operations to perform with the ServiceBinding:",
+		Options: []string{"create it on cluster", "display it", "save it to file"}, // respect order of CreateOption constants
```

Comment:
Which/where are these `CreateOption` constants that you talk about?

Reply:
I fixed the typo: it was CreationOption

Reply:
I fixed the typo: it was CreationOption (constants defined in the interface.go file of the same package, and of this type CreationOptions)